plugins/experimental/esi/lib/Variables.cc

```diff
@@ -357,9 +357,25 @@ Variables::_parseCookieString(const char *str, int str_len)
   AttributeList cookies;
   Utils::parseAttributes(str, str_len, cookies, ";,");
   for (AttributeList::iterator iter = cookies.begin(); iter != cookies.end(); ++iter) {
-    _insert(_dict_data[HTTP_COOKIE], string(iter->name, iter->name_len), string(iter->value, iter->value_len));
-    _debugLog(_debug_tag, "[%s] Inserted cookie with name [%.*s] and value [%.*s]", __FUNCTION__, iter->name_len, iter->name,
-              iter->value_len, iter->value);
+    std::string v = iter->name;
```

Comment:
The variable names are not chosen well.  Can you please not use 1 or 2 characters for variable names.